setup.env

```diff
@@ -63,6 +63,7 @@ arm64-nxp-ls1046ardb-r0
 arm64-delta-tx4810-r0
 arm64-delta-tn4810m-r0
 arm64-delta-tn48m2-r0
+arm64-delta-tn48m2-swdev-r0
```

Comment:
Since the only difference is the SPI flash size, please reconsider renaming the suffix from "swdev" to something like "spi64m". The fact that the board has extra SPI can be used for other things in the HW level, and it is not specific to switch-dev, which is in the software layer only.

Reply:
I agree this platform name isn't quite distinctive, however the name was decided by our customer.